ARCs/arc-0034.md

```diff
@@ -0,0 +1,63 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The purpose of this ARC is to set the parameters for proposals submitted by the community for evaluation by xGovs prior to a general governance vote for final approval.
```

Comment:
The abstract, the title and description are not coherent (i.e. they are talking about parameters, process and criteria for creation of a proposal), and do not cover all the questions in the rationale (which seem to be broader). It would be helpful to unify these parts to clarify what the ARC is really about.